Appraisals

```diff
@@ -38,3 +38,13 @@ appraise "rails6.0" do
   gem "spring-watcher-listen", "~> 2.0.0"
   gem "sqlite3", "~> 1.4"
 end
+
+appraise "rails6.1" do
+  gem "byebug"
+  gem "listen", "~> 3.2"
+  gem "puma", "~> 5.0"
+  gem "rails", "~> 6.1.0", ">= 6.1.0.0"
+  gem "spring"
```

Comment:
Ah, sorry I missed this in the last review, we will need to avoid the broken version of spring here too


```suggestion
  gem "spring", "!= 2.1.1"
```

Reply:
I'll go ahead and merge so we can get the tests running